### ISSUE TITLE
Restore the contextual menu for keyboard users (blind, etc)

### ISSUE THREAD

**Issue by alexarnaud** [hackademix/noscript#63]
Hello Noscript developer,

I'm helping multiple blind users to use their computer and I'm myself visual-impaired. They all have used NoScript on Firefox 52 without encounter any issues with only the keyboard (because they are blind and so can't the mouse), since the upgrade to Firefox ESR 60, the easy and practical contextual menu has been dropped and in consequence they are no longer able to use NoScript to protect them from Javascript on the web.  

The new menu is not keyboard navigable due to #29 and to be honest I think we'll spent more time to figure out how to make it usable with keyboard and screen reader instead of restoring the contextual menu.

Could it be possible to add the same contextual menu as before to make blind users able to use NoScript on Firefox ESR 60 or newer ?

Thanks in advance and thanks for your great work on NoScript.

Best regards,
Alex.

**Comment by fwiep** [comment]
Hello,
I am visually impaired myself and use NoScript on a daily bases. I would very much appreciate the ability to control NoScript by keyboard. Using Firefox 65 on Ubuntu 18.10.
Thank you very much!
Frans-Willem

**Comment by mhussaincov** [comment]
hi there,
I also would like this feature restored,
the ease and simplisity of the old contextual menu was so intuative and very accessible.
I would be greatful if this could be considdered.
if the old menu can't be reinstated, could it be made so when you use the applications key you can choose what element etc can be blocked?
under the noscript submenu?

thanks,
Majid

**Comment by vojtapolasek** [comment]
Hi, I would also gladly welcome back this feature. It makes things very easy.

**Comment by alexarnaud** [comment]
@hackademix Do you think we could have a change to make this issue fixed ?

Best regards,
Alex.

**Comment by hackademix** [comment]
No, unfortunately the old contextual menu cannot be restored as it was because of intrinsic limitations of the WebExtensions API.
On the other hand, we're currently working with Simply Secure, a firm specialized in User Experience, to drastically improve the accessibility of the new UI, including better keyboard navigation for the visually impaired and other users with special needs. Thanks for your feedback.
